Formula/kim-api-v2.rb

```diff
@@ -0,0 +1,34 @@
+class KimApiV2 < Formula
+  desc "The Knowledgebase of Interatomic Models (KIM) API"
+  homepage "https://openkim.org"
+  url "https://s3.openkim.org/kim-api/kim-api-v2-2.0.0.txz"
+  sha256 "beb9eb47cfe9ac9a4ad22528da89a67415a5b033075a9318a5d4a800dd4a3f3d"
+
+  depends_on "doxygen" => :build
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    system "make", "docs"
+    system "make", "install"
+
+    # compiler shims don't seem to work after install
+    ## Is there a better way that would get a "regular" full path the compilers
+    ## that are used to build the package?
+    inreplace "#{lib}/kim-api-v2/cmake/kim-api-v2-config.cmake" do |s|
+      s.gsub! /(CMAKE_CXX_COMPILER \")([^\"]+\/)([^\"\/]+)\"/, '\1\3"'
+      s.gsub! /(CMAKE_C_COMPILER \")([^\"]+\/)([^\"\/]+)\"/,   '\1\3"'
+    end
+
+    # although #{prefix} is not a "standard" one, we don't want/need
+    # the activate or deactivate scripts
+    remove "#{bin}/kim-api-v2-activate"
```

Comment:
If the files are installed by default, I suppose they are useful, why remove them?

Reply:
They are only installed when the package is installed in a place that is likely to not be on the path (a subdirectory of user's home dir, etc.).  Then these scripts add the package to the path and other similar things that are not necessary (are automatic) when the package is installed in a standard prefix (/usr, /usr/local).

In some sense might be better to build the package with <prefix> equal to /usr/local and then use DESTDIR to install it to the real prefix.  Is that something that is commonly done with homebrew?

Reply:
Nope. Let's keep it like that, but maybe shorten the comment a bit.